client/apps/settings/client/curations/display/components/campaign.jsx

```diff
@@ -8,6 +8,15 @@ function onChangeDate (name, value, index, onChange) {
   onChange(name, date, index)
 }
 
+function setDefaultSov (value) {
+  var newValue = value.toString()
+  if (newValue.length === 3) {
+    // add a trailing 0 to .5
+    newValue = newValue + '0'
+  }
+  return newValue
+}
```

Comment:
This makes me wonder why we are saving these values as strings and not floats. They could even be integers with a max of 100. Appending '0' to a string when later we will later want to support things like SOV sum validation seems like we'd regret this pretty quickly imo.

Reply:
We are saving as floats, it just needs to be a string to render the options on the select

Reply:
Ah I see, my bad! Well in that case I think [toFixed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) solves the string problem which _should_ remove the need for this method.

Reply:
This is actually going the other way around, toString is the desired result.  Turning .75 to string works fine, but .5 returns '0.5' rather than '0.50'.  The select can't render a saved value unless it is an exact match.  The value is saved as an integer here if saved - https://github.com/eessex/positron/blob/7c8dc119a5a452ac68a3b35d0b4142357986fa15/client/apps/settings/client/curations/display/components/campaign.jsx#L54

Reply:
oh wait i misunderstood -- yes will use toFixed